etc/afpd/status.c

```diff
@@ -499,7 +499,7 @@ void status_init(AFPObj *dsi_obj, AFPObj* asp_obj, DSI *dsi)
     status_machine(status);
     status_versions(status,
 #ifndef NO_DDP
-                    asp,
+                    NULL,
 #endif
```

Comment:
Can we remove the ifdef and always send the NULL parameter now? That would allow us to simplify `status_versions()` as well, unless I'm mistaken.

Reply:
Not easily. The ASP struct that is defined in `status_versions()` doesn't exist if NO_DDP is set.

Reply:
Ah ok, maybe a rafactoring for another time. It would be nice to reduce the `#ifndef NO_DDP` spaghetti and use normal logic.